Sources/StreamChat/Database/StreamChatModel.xcdatamodeld/StreamChatModel.xcdatamodel/contents

```diff
@@ -253,6 +240,7 @@
         <attribute name="isOnline" attributeType="Boolean" usesScalarValueType="YES"/>
         <attribute name="lastActivityAt" optional="YES" attributeType="Date" usesScalarValueType="NO"/>
         <attribute name="name" optional="YES" attributeType="String"/>
+        <attribute name="teams" optional="YES" attributeType="Transformable" valueTransformerName="NSSecureUnarchiveFromData"/>
```

Comment:
Let's also specify the custom type just to be explicit about the data we store

<img width="259" alt="Screenshot 2021-11-12 at 13 17 04" src="https://user-images.githubusercontent.com/12818985/141450776-7ca9661c-32d2-4d48-b4b6-9d76f7c13224.png">

Reply:
good catch